HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -71,6 +71,18 @@ def customiseFor18330(process):
           for pset in producer.vertexCollections:
                if pset.algorithm.value() == "AdaptiveVertexFitter" and not hasattr(pset, "chi2cutoff"):
                     pset.chi2cutoff = cms.double(3.0)
+     return process
+
+
+# Add new parameters required by RecoTauBuilderConePlugin
+def customiseFor18429(process):
+     for producer in producers_by_type(process, "RecoTauProducer"):
+          if hasattr(producer,'builders'):
+               for pset in producer.builders:
+                    if not hasattr(pset,'minAbsPhotonSumPt_insideSignalCone'):
```

Comment:
@davidlange6 Yes, in principle fillDescriptions will solve this and similar issues caused by the fact that HLT configuration is dumped from ConfDB which knows on new parameters only when a new release is parsed into it. The thing is that the tau reco code uses quite heavily plugin factories which, at least in my understanding, are quite incompatible with the fillDescriptions mechanism. So mitigation from python cfi files to the fillDescriptions mechanism means rewriting tau code which is a big effort.